aptos-move/framework/src/module_metadata.rs

```diff
@@ -57,6 +64,8 @@ pub struct KnownAttribute {
 #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize)]
 pub enum KnownAttributeKind {
```

Comment:
Unfortunately the choice of enum may cause a problem, namely if we do rollbacks as recently. Because then some new data is written somewhere but after rollback can't be deserialized. This is a new dimension of compat we have not considered. Not for this PR but to remember.